G/GMT/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GMT"
+version = v"6.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/GenericMappingTools/gmt/releases/download/$(version)/gmt-$(version)-src.tar.gz", "ab7062912aeead1021770fad4756e0a99860fde8ea9b428fb00c22fa15a3bbfc")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd gmt-6.2.0/
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DHAVE_QSORT_R_GLIBC=False -DHAVE___BUILTIN_BSWAP16=False -DHAVE___BUILTIN_BSWAP32=False -DHAVE___BUILTIN_BSWAP64=False   .. 
+make -j${nproc} 
+make install 
+exit
```

Comment:
```suggestion
cd $WORKSPACE/srcdir/gmt-*
mkdir build
cd build/
cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release \
    -DHAVE_QSORT_R_GLIBC=False \
    -DHAVE___BUILTIN_BSWAP16=False \
    -DHAVE___BUILTIN_BSWAP32=False \
    -DHAVE___BUILTIN_BSWAP64=False \
    .. 
make -j${nproc} 
make install 
```
is a bit more readable (and independent from upstream version)